src/Documentation/doc/advanced/bitnode_recommendation.md

```diff
@@ -0,0 +1,240 @@
+
+## BitNode 5
+
+This BitNode unlocks:
+
+- [Intelligence](./intelligence.md): It's a permanent stat that buffs many things.
+- BitNode's multiplier data in UI and NS APIs.
+- Permanent access to `Formulas.exe`. Formulas APIs are useful for many mechanics, not just hacking. Having free access to them is a very good benefit.
+
+Source-File 5 also buffs hacking-related multipliers. Hacking is a core mechanic, so it's always good to have higher hacking-related multipliers.
+
+## BitNode 6 and BitNode 7
+
+These BitNodes unlock [Bladeburner](./bladeburners.md). Bladeburner gives you an alternative way to destroy WD. In Bladeburner, you can do many actions (General, Contracts, Operations, Black Operations) to get money, experience, Bladeburner's rank, etc. You can destroy WD after completing the last Black Operation.
+
+Bladeburner is a slow mechanic, but it's rarely nerfed in other BitNodes. Even when it's nerfed, the nerf is not too severe. This is why Bladeburner is a good choice in extremely hard BitNodes (e.g., BitNode 9, BitNode 13).
+
+Both BitNodes grant Bladeburner access outside these BitNodes. The differences are:
+
+- BitNode 6 does not have Bladeburner's penalty modifiers. Its Source-File buffs combat stats' multipliers.
+- BitNode 7 has Bladeburner's penalty modifiers. Its Source-File buffs Bladeburner's multipliers. Source-File 7.3 gives you free access to "The Blade's Simulacrum" augmentation. You will immediately receive this augmentation after joining the Bladeburner division.
+
+When you are performing Bladeburner action, you cannot do other actions (working, committing crimes, etc.). It's a downside of Bladeburner. "The Blade's Simulacrum" augmentation removes this restriction by allowing you to perform Bladeburner actions and other actions at the same time.
+
+This mechanic is time-gated by the slow generation speed of contracts/operations. Sleeves speed it up a lot.
+
+You must be careful with chaos and Synthoid population. The UI shows you many hints about the effect of dangerous actions on chaos and Synthoid population. You should keep an eye on those hints. Generally, you must keep the chaos level low and not kill too much Synthoid population.
+
+## BitNode 8
+
+This BitNode unlocks 2 new features for the stock market: short stock and limit/stop order.
+
+This is a very challenging BitNode. It disables most normal ways to earn money so that you have to use the Stock Market as your source of income. Before having access to 4S Market Data, finding good stocks to invest in is pretty hard. You need to have a good pre-4S stock market script; otherwise, it's a slow run. You should take a look at stock manipulation. Hacking does not give you money, but hacking and growing a server that has a corresponding stock still influence that stock's price.
+
+Grafting is very useful in this BitNode. Losing capital after each resetting sets you back a lot. Grafting allows you to continuously install augmentations without resetting.
```

Comment:
Casino is the primary fund pool for getting started in BN-8. Is it worth mentioning here?